src/DynamoCore/Configuration/Configurations.cs

```diff
@@ -116,6 +116,11 @@ public class Configurations
         /// </summary>
         public static readonly double ZoomIncrement = 0.05;
 
+        /// <summary>
+        /// Zoom Threshold Level 
+        /// </summary>
+        public static readonly double ZoomLevel = 0.4;
```

Comment:
Maybe use `ZoomThreshold` as it is clearer just like `ZoomIncrement` above?

Reply:
Thanks, Aaron! Naming is hard ^^